shp2osm/taglist.csv

```diff
@@ -1,99 +1,97 @@
+welder / metal worker,craft,welder,,,,,https://wiki.openstreetmap.org/wiki/Tag:craft%3Dwelder
+tailor / dress maker,shop,tailor,,,,,https://wiki.openstreetmap.org/wiki/Tag:shop%3Dtailor
+car repair / mechanic,shop,car_repair,,,,,https://wiki.openstreetmap.org/wiki/Tag:shop%3Dcar_repair
+vulcanizer / tyre repair,shop,tyres,,,,,https://wiki.openstreetmap.org/wiki/Tag:shop%3Dtyres
+carpenter / wood worker,craft,carpenter,,,,,https://wiki.openstreetmap.org/wiki/Tag:craft%3Dcarpenter
+Boat builder,craft,boatbuilder,,,,,https://wiki.openstreetmap.org/wiki/Tag:craft%3Dboatbuilder
+land used for a school,amenity,school,,,,,https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dschool
+Any building,building,yes,,,,,https://wiki.openstreetmap.org/wiki/Key:building
+A building with healthcare of unknown purpose (not a shop selling medicine) ,healthcare,centre,,,,,https://wiki.openstreetmap.org/wiki/Tag:healthcare%3Dcentre
+Dispensary selling medicine,amenity,pharmacy,,,,,https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dpharmacy
+telecommunication mast (man-made),man_made,mast,tower:type,communication,,,https://wiki.openstreetmap.org/wiki/Tag:man_made%3Dmast
+Restaurant,amenity,restaurant,,,,,https://wiki.openstreetmap.org/wiki/Tag:amenity%3Drestaurant
+bar(drinks only),amenity,bar,,,,,https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dbar
+Barber shop,shop,hairdresser,male,yes,,,https://wiki.openstreetmap.org/wiki/Tag:shop%3Dhairdresser
+beauty salon,shop,hairdresser,female,yes,,,https://wiki.openstreetmap.org/wiki/Tag:shop%3Dhairdresser
+Internet cafe,amenity,internet_cafe,,,,,https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dinternet_cafe
+Photograph shop,shop,photo,,,,,https://wiki.openstreetmap.org/wiki/Tag:shop%3Dphoto
+Motorcycle services,shop,motorcycle,,,,,https://wiki.openstreetmap.org/wiki/Tag:shop%3Dmotorcycle
+Frozen yoghurt shop,amenity,ice_cream,,,,,https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dice_cream
+Convenience shop/mini-mart,shop,convenience,,,,,https://wiki.openstreetmap.org/wiki/Tag:shop%3Dconvenience
+fuel shop (They are often just a house with some barrels or bottles clearly displayed.),shop,fuel,,,,,https://wiki.openstreetmap.org/wiki/Tag:shop%3Dfuel
+Fuel station,amenity,fuel,,,,,https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dfuel
+A building that is a shed,building,shed,,,,,https://wiki.openstreetmap.org/wiki/Tag:building%3Dshed
+A building that has no walls,building,roof,,,,,https://wiki.openstreetmap.org/wiki/Tag:building%3Droof
+community Hall/Center,amenity,community_centre,,,,,https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dcommunity_centre
+place of worship,amenity,place_of_worship,,,,,https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dplace_of_worship
+Hotel,tourism,hotel,,,,,https://wiki.openstreetmap.org/wiki/Tag:tourism%3Dhotel
+ Motel,tourism,motel,,,,,https://wiki.openstreetmap.org/wiki/Tag:tourism%3Dmotel
+ Guest house,tourism,guest_house,,,,,https://wiki.openstreetmap.org/wiki/Tag:tourism%3Dguest_house
```

Comment:
For "Motel" and "Guest house" there is a blank space before the "Name". Will it affect processing?
@jeafreezy

Reply:
Not at all, it won't